analysis/paper/paper.Rmd

```diff
@@ -46,7 +46,7 @@ Putative “doublet” cell barcodes were identified and removed from downstream
 ```{r echo = FALSE, message=FALSE}
```

Comment:
Author list: note what the * means, remove final comma

Abstract: Replace with actual abstract from the final manuscript

Reply:
Comma removed and '*' explicitly defined below.

Abstract replaced with Abstract in the final manuscript